feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/components/TimetableItemDetailHeadline.kt

```diff
@@ -66,8 +68,10 @@ fun TimetableItemDetailHeadline(
         FlowRow {
             TimetableItemTag(
                 modifier = Modifier.align(Alignment.CenterVertically),
-                tagText = timetableItem.room.name.currentLangTitle,
-                tagColor = roomTheme.primaryColor,
+                tagText = timetableItem.room.name.currentLangTitle.toUpperCase(Locale.current),
+                contentTextColor = MaterialTheme.colorScheme.surface,
+                contentBackgroundColor = roomTheme.primaryColor,
+                borderColor = roomTheme.primaryColor,
                 icon = timetableItem.room.icon,
             )
```

Comment:
The TimetableItemRoomTag was not in a format that could be referenced from here. 🥹